### DISCUSSION TITLE
Simulated dbm

### DISCUSSION THREAD

**Discussion by cin89** [aligungr/UERANSIM#499] [Q&A]
Hi @aligungr ,

Could you explain how Heartbeat mechanism works? Simulated dbm is supposed to be reported for each heartbeat_ack message. However I don't fully understand how phylocation from GnbConfig struct is used. Is this field taken from config file? How to trigger signal lost from UE perspective?

Best Regards

**Comment by aligungr** [comment]
Hi @cin89 

Physical location exists in the code base but actually it's not used. It was planned to be used in the future.
So currently there is only one way to trigger a signal lost, that is, heartbeat failure.
So, if you want to trigger a signal lost, you can kill the UE or gNodeB processes for example. Or you can drop the heartbeat packets etc.

Best.
